doc/src/learn/architecture/consensus.md

```diff
@@ -75,15 +78,15 @@ flowchart TB
 * The certificates prove the data availability of each collection, or block, at each round.
 * Their contents constitute a DAG that can be traversed identically at each honest node.
 
-While the Bullshark consensus selects a specific DAG traversal among several a posteriori, both Bullshark and external consensus algorithms can add more sophistication to their selection of blocks / collections to reflect priority concerns.
+While the Bullshark or Tusk consensus selects a specific DAG traversal among several a posteriori, both they and external consensus algorithms can add more sophistication to their selection of blocks / collections to reflect priority concerns.
 
 ## Dependencies
 
 Narwhal is implemented using [Tokio](https://github.com/tokio-rs/tokio), [RocksDB](https://github.com/facebook/rocksdb/) and generic cryptography. The cryptography contains implementations of node signing using BLS12-377, BLS12-381, and Ed25519.
 
 ## Configuration
 
-To conduct a fresh deployment of Narwhal and Bullshark, follow the instructions at [Running Benchmarks](https://github.com/mystenlabs/narwhal/tree/main/benchmark).
+To conduct a fresh deployment of Narwhal and either Bullshark or Tusk, follow the instructions at [Running Benchmarks](https://github.com/mystenlabs/narwhal/tree/main/benchmark).
```

Comment:
To deploy a new instance?

Reply:
@huitseeker , here is another question on your text.

Reply:
I'm not opposed to either formulation, but I have a small preference for having the complex bundle of names be a direct object, rather than the object of a preposition. It appears simpler to me.

Reply:
Solved with "Sui Consensus Engine"